GuildedRoseLLC/Items/ItemParser.swift

```diff
@@ -5,7 +5,6 @@ public class ItemParser {
     
     public static func parse(json: Data) -> [Item] {
         let decoder = JSONDecoder()
-        
         if let itemsList: [Item] = try? decoder.decode ([Item].self, from: json) {
```

Comment:
Can `[Item].self` be simplified to just `[Item]`?

Reply:
It cannot, it is referring to the class